### ISSUE TITLE
ever sence the jan 15 update

### ISSUE THREAD

**Issue by smd111** [Windower/Issues#674]
ever sence the jan 15 update windower will randomly load only some of the plugins addons that i want running
this is what should load
```

load LuaCore;
load AutoExec;
load ChatMon;
load StatusTimer;
lua load cancel;
lua load ConsoleBG;
lua load distance;
lua load DressUp;
lua load GearSwap;
lua load pointwatch;
lua load TParty;
lua load Treasury;
```
but what loads is this
```
load LuaCore;
load StatusTimer;
lua load cancel;
lua load ConsoleBG;
lua load distance;
```
this is random 
and even if i do load thoes that do not auto load it just causes the plugins/addons not to work correctly

i cant try to capture info with windgb but i wont be able to do it till next weekl

**Comment by z16** [comment]
Can you check if you launch Windower, after logging into POL, before logging into FFXI, if `Windower/scripts/autoload/autoload.txt` contains all the correct plugins/addons?

**Comment by sharinganheir** [comment]
This sounds exactly like what has been going on for a year that I mentioned in a forum thread, but I've never heard of anyone else having this exact issue til now. If it helps shed some light on the issue, I looked at my files in relation to the question and autoload.txt has not been modified since March of last year. I do use profiles to login and the profile file contains all the correct plugins/addons. 

Addenum: I just tried with default which did update autoload.txt, but still hung up on the addons after the second addon loaded. Whenever attempting to shutdown to try again makes windower stop responding each time.  Often loading or unloading plugins or addons causes a crash, and when it doesn't plugins/addons still do not function correctly.
